taigit/src/libraries/Taiga.ts

```diff
@@ -498,4 +498,29 @@ taiga_issues(projId : number) : Promise<Object>{
         output.push(new_entry);
     }
     return output;
-}
\ No newline at end of file
+}
+
+/**
+ * @summary This call return sprint velocity (unit points) based on sprint Id
+ * @param sprintId the ID for the access of sprint stats
+ * @returns
+ * * {
+ *          velocity : number //sprint velocity
+ * }
+ */
+export async function
+sprint_velocity_pts(sprintId : number) : Promise<Object> {
+    let data = await axios.get("https://api.taiga.io/api/v1/milestones/"+sprintId.toString()+ '/stats');
+    //test id:  https://api.taiga.io/api/v1/milestones/205316/stats
+    //https://api.taiga.io/api/v1/milestones/219984/stats
+
+     let entry = {
+            pts : data.data.completed_points
+        }
+    let velocity : number = 0;
+    for(let each of entry.pts ){
+        velocity += each;
```

Comment:
Does this just add up the total points? Shouldn't the velocity be `total number of points completed / total points` ?

Reply:
For a single sprint velocity, it is the completed points.